odoo/addons/base/data/res_currency_rate_demo.xml

```diff
@@ -727,10 +727,10 @@
             <field name="rate">1105.24376765</field>
         </record>
 
-        <record forcecreate="0" id="rateZWL" model="res.currency.rate">
-            <field name="currency_id" ref="ZWL" />
+        <record forcecreate="0" id="rateZIG" model="res.currency.rate">
+            <field name="currency_id" ref="ZIG" />
             <field name="name">2010-01-01</field>
-            <field name="rate">395.80</field>
+            <field name="rate">13.56</field>
```

Comment:
13.56 is against the USD, but we must be giving the rate against the currency with rate 1, which is the Euro. :)

Reply:
Indeed.
 based on the (exchange rate document)[https://www.rbz.co.zw/documents/Exchange_Rates/2024/April/RATES_8_APRIL_2024.pdf], Mid rate for Euro is 14.69.